pygmt/tests/test_datasets_samples.py

```diff
@@ -186,9 +186,11 @@ def test_rock_sample_compositions():
     """
     data = load_sample_data(name="rock_compositions")
     assert data.shape == (1000, 3)
+    assert data["limestone"].min() == 0
+    assert data["limestone"].max() == 1
```

Comment:
After adding the `permittivity` there are four columns.

```suggestion
    data = load_sample_data(name="rock_compositions")
    assert data.shape == (1000, 4)
    assert data["limestone"].min() == 0
    assert data["limestone"].max() == 1
```